gapis/api/cmd_id_group.go

```diff
@@ -307,24 +322,27 @@ func (g CmdIDGroup) IterateBackwards(index uint64, cb func(childIdx uint64, item
 // All groups must be added before commands.
 // Attemping to call this function after commands have been added may result in
 // panics!
-func (g *CmdIDGroup) AddGroup(start, end CmdID, name string) error {
+func (g *CmdIDGroup) AddGroup(start, end CmdID, name string) (*CmdIDGroup, error) {
```

Comment:
F2F discussion: These changes are also done by one of my CLs. I will put this change up for review ASAP.

Reply:
Not yet updated, will wait for rebase